ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/electra/block/BlockProcessorElectra.java

```diff
@@ -240,7 +279,70 @@ public void processDepositRequests(
     }
   }
 
-  /** Implements process_withdrawal_request from consensus-specs (EIP-7002 & EIP-7251). */
+  /*
+   * <spec function="process_withdrawal_request" fork="electra">
+   * def process_withdrawal_request(
+   *     state: BeaconState,
+   *     withdrawal_request: WithdrawalRequest
+   * ) -> None:
```

Comment:
this is another good example of it just being too much maybe...

Reply:
This is a tough one. It's just a really large function in the specs.